_pages/author-adarsh.html

```diff
@@ -0,0 +1,28 @@
+---
+title: "Adarsh Singh"
+layout: default
+permalink: "/author-adarsh-singh"
```

Comment:
`permalink: "/author-adarsh-singh"` should match with the filename

## Changed that you can make 
1. Either make it `permalink: "/author-adarsh"` 
2. or change the filename to  `_pages/author-adarsh-singh.html`